common/src/ess.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+use std::{ops::Deref, sync::Arc, time::SystemTime};
+
+use crate::proto::{
+    common::Value as ValueMessage,
+    common::{value::Value as ValueEnum, SubscribeFulfillment, SubscribeIntent},
+    streaming::{channel_service_server::ChannelService, Event, OpenRequest},
+};
+use async_trait::async_trait;
+use ess::EventSubSystem;
+use tokio::spawn;
+use tokio_stream::wrappers::ReceiverStream;
+use tonic::{Response, Status};
+use uuid::Uuid;
+
+type InnerEss<T> = EventSubSystem<Box<str>, Box<str>, T, Result<Event, Status>>;
```

Comment:
Can we rename `InnerEss` to `Ess` or (preferably) `EventSubSystem`? I don't think the _inner_ labeling is right. This is just specifying some type arguments and leaving one open to parameterisation.

Reply:
Done with d7deb59057a7f2d53035d6f13df0bed7775f0223.